src/status_im/subs.cljs

```diff
@@ -404,6 +404,11 @@
 
 ;;GENERAL ==============================================================================================================
 
+(re-frame/reg-sub
+ :ui/new-enabled?
+ (fn [_]
+   @config/new-ui-enabled?))
```

Comment:
i don't think its reactive, so you should use it directly without subscription

Reply:
I did try this, but it doesn't work @flexsurfer. The subscription really needs to return a boolean, otherwise `(<sub ...`) always evaluates to true.

Reply:
i mean you should use `@config/new-ui-enabled?` directly in the view

Reply:
I see @flexsurfer, I just wanted to decouple the UI from having to know where the state comes from. It could also be injected as a coeffect, but if you prefer the views to consume the state directly from the atom that's fine by me. I assumed you would be okay with this implementation because the `:home-items` subscription already uses `config/new-ui-enabled?`.

You might also be interested in an apparent bug in the `:home-items` subscription, related to this very discussion about the `config/new-ui-enabled?` atom (https://github.com/status-im/status-mobile/pull/13747#discussion_r935414648).

Reply:
`@config/new-ui-enabled?` used only for development, it's a temporary flag which will be removed when we have new design implemented

Reply:
https://github.com/status-im/status-mobile/pull/13747/commits/79cb56a15be5a126bb0f4ee50fd9ba60b5297d2e